modules/core/04-channel/keeper/upgrade.go

```diff
@@ -181,7 +181,7 @@ func (k Keeper) ChanUpgradeTry(
 
 // WriteUpgradeTryChannel writes the channel end and upgrade to state after successfully passing the UpgradeTry handshake step.
 // An event is emitted for the handshake step.
-func (k Keeper) WriteUpgradeTryChannel(ctx sdk.Context, portID, channelID string, upgrade types.Upgrade, upgradeVersion string) (types.Channel, types.Upgrade) {
+func (k Keeper) WriteUpgradeTryChannel(ctx sdk.Context, portID, channelID string, upgrade types.Upgrade, upgradeVersion string, counterpartyLastPacketSequence uint64) (types.Channel, types.Upgrade) {
```

Comment:
small nit: should we make the args consistent across both write funcs. maybe `counterpartyLastSequenceSend`

Reply:
we most certainly can!